src/status_im/android/platform.cljs

```diff
@@ -37,8 +37,7 @@
                                       :background-color "#7099e619"
                                       :border-radius    5
                                       :padding          4}
-                           :item     {:status-text {:color       styles/color-black
```

Comment:
@rasom Hi Roman, thanks for your quick review!
The reason is that `line-height` not works very well with `color` on Android.
I just set up a minimal example using Expo: 
https://github.com/tiensonqin/text-style-example/blob/master/src/text_style_example/core.cljs#L10